source/includes/steps-source-command-palette.yaml

```diff
@@ -0,0 +1,40 @@
+---
+title: "In |vscode-short|, open the Command Palette."
+level: 4
+ref: vsce-command-palette
+content: |
+
+  .. include:: /includes/list-tables/command-palette-options.rst
+---
+title: "From the Command Palette, choose where you want to save the 
```

Comment:
This is not a thing anymore. We found that the UX was a bit confusing so we skip this step now. In the settings of the extension, users can configure what they want the behavior to be.

![image](https://user-images.githubusercontent.com/761042/81658783-619dce00-9439-11ea-8345-85c5fe5923e2.png)

Reply:
@mmarcon, I removed the steps from both tabs and called out the setting in the intro to the [Create a Connection](https://docs-mongodbcom-staging.corp.mongodb.com/5d31072/mongodb-vscode/docsworker/DOCSP-9711/connect#create-a-connection-to-a-deployment) section.